src/google-maps/google-map/google-map.ts

```diff
@@ -258,22 +276,19 @@ export class GoogleMap implements OnChanges, OnInit, OnDestroy {
     if (this._isBrowser) {
       this._mapEl = this._elementRef.nativeElement.querySelector('.map-container')!;
       this._setSize();
-      this._googleMapChanges = this._initializeMap(this._combineOptions());
```

Comment:
I don't remember exactly why we were re-creating the map whenever the options change, considering that the map object has a `setOptions` which we were calling. I've removed it since the demo was working as expected and all the tests passed.

Reply:
It doesn't recreate the map. The initializeMap method has a "take(1)" operator which means it only takes in the first options config. So it then relies on setOptions after that. Why was it set up that way? It made more sense when getting the initial options required combining a lot of BehaviorSubjects. I think this new way should be fine.